server/migrations/20211117015845-rank-search-results.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+
+      await queryInterface.sequelize.query(
+        'UPDATE "OffchainThreads" SET _search = ' +
+        '= (setweight(to_tsvector(\'english\', coalesce(title, \'\')), \'A\') || ' +
```

Comment:
There an extra redundant '=' here? I get an error when attempting to migrate.

Reply:
Fixed